src/components/ActNowSummary/index.js

```diff
@@ -0,0 +1,56 @@
+import { Grid, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: "flex",
+    backgroundColor: "#F7F7F7",
+    padding: "1rem",
+    justifyContent: "space-around",
+    alignItems: "flex-end",
+  },
+  count: {
+    fontSize: "1.2rem",
+    fontWeight: 700,
+  },
+}));
+
+function ActNowSummary({ className, summary, ...props }) {
+  const classes = useStyles(props);
+  return (
+    <Grid container className={classes.root}>
+      <Grid item xs={2}>
+        <Typography variant="h4" className={classes.count}>
+          {summary.accounts.count}
+        </Typography>
+      </Grid>
+      <Grid item xs={4}>
+        <Typography variant="body2" noWrap>
```

Comment:
```suggestion
        <Typography variant="h4" noWrap>
```
![Screenshot from 2021-07-23 09-08-42](https://user-images.githubusercontent.com/1779590/126743450-d9151535-0c54-4f98-bbf5-baf366a7a512.png)